lua/epics.lua

```diff
@@ -111,6 +111,25 @@ function M.setup(opts)
 
 	M.options = vim.tbl_extend("force", M.options, opts)
 
+	vim.filetype.add {
+		extension = {
+			ioc = "make",
+			Db = "make",
+		},
```

Comment:
Do you know of `.ioc` and `.Db` makefiles other than `RULES.ioc` and `RULES.Db`? If not I think it's best to remove these extensions.

Reply:
IMHO, it is totally ok to remove it.
But maybe someone wants to copy some backups like `bak_RULES.ioc` or other names.

Reply:
The thing that worries me is mostly the `.Db` file, if someone uses that extension for database files. In the end neither extension are Makefile extensions. I personally think that thoses files should have been named `RULES_IOC` and `RULES_DB` but oh well.

Reply:
@minijackson I tend to agree; those files were named over 20 years ago. Unfortunately a reference to the filename `$(CONFIG)/RULES.ioc` probably appears in almost every EPICS IOC application's `configure/RULES.ioc` file at every EPICS site, so that file is never likely to get changed, but it looks like we might be able to rename `RULES.Db` as that is directly included only by the Base `configure/RULES_BUILD` file.

Reply:
@konosubakonoakua in the end I think it's best to remove them, I think with the filenames and patterns we should cover most cases.

@anjohnson is the rename of `RULES.Db` something that epics-base plan on doing in the near future?

Reply:
Done~

Reply:
@minijackson I see it as a minor thing which I might do when I'm next working on the build rules, but that might be some time off (months or more) and I can't guarantee that I'd remember. If you or @konosubakonoakua were to create a PR against Base for that it would probably get accepted and merged sooner, although new Base versions are normally released near the end of the calendar year.

If only my preferred editor ([nedit-ng](https://eteran.github.io/nedit-ng/latest/)) had full filename recognition of file types, it's configurable but only with extensions...